microsoft-edge/extensions-chromium/publish/aad-account.md

```diff
@@ -20,7 +20,7 @@ If you do not have an account and are new to Partner Center, continue to Step **
 1. If you are new to Partner Center and don't have an account or if you have an existing commercial account with Partner Center (see step **2.c** below), and for more information navigate to [Microsoft account (MSA)][WindowsCommunityEverythingAboutMicrosoftAccounts] (or use an existing one) to register with the Microsoft Edge program. Follow the steps at [Register as a Microsoft Edge extensions developer][DeveloperRegistration]. 
```

Comment:
optional/discretionary: update the date at top of file.  The Date on these articles means the last time the article was reviewed and confirmed to be reasonably accurate & up-to-date, or the last time the article was significantly updated.